functionnal_tests/cypress/integration/app_file/translation.js

```diff
@@ -0,0 +1,22 @@
+import { PAGES } from '../../support/urls_commands.js'
+
+describe('App File', () => {
+  before(() => {
+    cy.resetDB()
+    cy.setupBaseDB()
+    cy.loginAs('users')
+    cy.fixture('baseWorkspace').as('workspace').then(workspace => {
+      cy.visit(`/ui/workspaces/${workspace.workspace_id}/contents?type=file`)
```

Comment:
There is not this URL in the PAGES variable? If not, do you see an obstacle to adding it? (If you decide not to add it, there is an `import` not used in this file.)

Reply:
This applies to all similar cases.